homeassistant/components/modbus/base_platform.py

```diff
@@ -86,13 +90,30 @@ def __init__(self, hub: ModbusHub, entry: dict[str, Any]) -> None:
     async def async_update(self, now=None):
         """Virtual function to be overwritten."""
 
-    async def async_base_added_to_hass(self):
-        """Handle entity which will be added."""
+    async def async_remote_start(self) -> None:
+        """Remote start entity."""
         if self._scan_interval > 0:
-            cancel_func = async_track_time_interval(
+            self._cancel_timer = async_track_time_interval(
```

Comment:
Do we need to cancel a potential running timer here first?

Reply:
no start cannot be called twice (the command from outside is restart, and it calls stop first).

Reply:
I don't see where we call stop first.

Reply:
Now we do.

It was in the old restart code (modbus.py 200+), but dropped out with the change to dispatcher, however it is more clear to have it in base_platform.py.